service/evidently/api_op_UpdateExperiment.go

```diff
@@ -64,6 +64,9 @@ type UpdateExperimentInput struct {
 
 	// Removes a segment from being used in an experiment. You can't use this parameter
 	// if the experiment is currently running.
+	//
+	// Deprecated: Use Boolean instead, and add the @default trait to structure members
+	// that targets this shape
 	RemoveSegment bool
```

Comment:
This blurb about `Deprecated` is really odd and doesn't make sense from a user of the SDK's point of view. From the wording this looks like some model or Smithy 2.0 related issue that shouldn't be present on public documentation of a member.

Reply:
I originally thought this diff was acceptable, but looking at the other smithy-* repos, it looks like that these comments are suppressed for Prelude shapes, e.g. in awslabs/smithy-typescript#581.

Reply:
Fixed in https://github.com/aws/smithy-go/pull/376/commits/5a226b6c3d049a0f8b7c9c5eae513981d8b0d680#diff-8a690a7a6321f66f8c1bf14b573baecaa233b532129c36d51ae6e301e36691e9